todo_app/app.py

```diff
@@ -80,14 +99,16 @@ def item_status():
     def del_item():
         del_title = request.form['del_title']
         delete_item(del_title)
+        logger.info("Item deleted: %s", del_title)
         return redirect(url_for('index'))
     
     # Route for logout
     @app.route('/logout')
-    @reader_required
+    @login_required
     def logout():
         logout_user()
-        return redirect(url_for('index'))
+        logger.info("User logged out. User id: $s", current_user)
```

Comment:
Not sure you've tested this - there's a slight typo, (`$` instead of `%`), the `current_user` object doesn't print in a very friendly manner, and the current_user is empty by the time this log is hit because they were signed out the line before!

Currently this prints:
```
Message: 'User logged out. User id: $s'
Arguments: (<flask_login.mixins.AnonymousUserMixin object at 0x000002201D305880>,)
```
We might want to tweak this to:
```suggestion
        logger.info("User logged out. User id: %s", current_user.id)
        logout_user()
```